samples/nrf9160/aws_iot/src/main.c

```diff
@@ -400,15 +400,12 @@ static void modem_configure(void)
 {
 	int err;
 
-	if (IS_ENABLED(CONFIG_LTE_AUTO_INIT_AND_CONNECT)) {
```

Comment:
`lte_lc_init_and_connect` can be called directly instead of `modem_configure`. It basically just wraps a single function.